ReactNativeClient/lib/services/CommandService.ts

```diff
@@ -243,6 +243,12 @@ export default class CommandService extends BaseService {
 		return command.runtime.title(command.runtime.props);
 	}
 
+	label(commandName:string):string {
+		const command = this.commandByName(commandName);
+		if (!command) return null;
```

Comment:
Could you make it throw an error here instead of returning null? Normally all commands must be declared early and it's not declared, it means there's a bug somewhere which should be fixed.

Reply:
I've added a throw to the function.